### PR TITLE
Added more frameworks

### PR DESCRIPTION
**Pull request by shashwat1998**
Added names of some more frequently used frameworks.

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/javascript/index.md [hunk 1]
```diff
@@ -28,4 +28,6 @@ Chakra, a fork of Jscript, is developed by Microsoft and used in their Edge brow
 
 Mozilla also offers Rhino, an implementation of JavaScript built in Java, typically embedded into Java applications to provide scripting to end users.
 
-WebKit (except for the Chromium project) implements the JavaScriptCore engine.
\ No newline at end of file
+WebKit (except for the Chromium project) implements the JavaScriptCore engine.
+ 
+Other most frequently used JavaScript Frameworks are React JS, Angular JS, Create JS, jQuery etc.
```
Comment (Asjas) [inline src/pages/javascript/index.md]: Hi @shashwat1998 

You are including JavaScript frameworks in the JavaScript Engines. Can you please remove it as JS Engines and JS Frameworks are not the same thing?

**Comment by Asjas** [comment]
Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :tada:

**Comment by shashwat1998** [comment]
@Asjas can you tell me more about those two automatic checks one is deploy check and the other. What are they?

**Comment by Asjas** [comment]
Travis CI is a Continuous Integration check that runs our tests.

Netlify deploys the website to see if it is working. There is currently a problem with the Netlify builds that we are aware of so it's being ignored.

**Pull request merged**